Problems/2606/ans_2606_YH.py

```diff
@@ -0,0 +1,73 @@
+#https://www.acmicpc.net/problem/2606
+from collections import deque
+
+# bfs로 풀기
+def bfs(node, graph, visited):
+    queue = deque([node])
```

Comment:
```suggestion
    queue = deque()
    queue.deque(node)
```
난 예시와 같이 쓰는데. deque initialize 시 node값을 넣어서 시작하고 싶었던 것 같긴 한데,
Iterable 로 만들기 위해 node라는 Integer를 굳이 1개짜리 list로 만들면서 생성하는 것이 좀 생경해보였음.
애당초 deque 생성 시 왜 인자가 iterable 을 받도록 설계되었는지, 설계의도 생각해보면 좋을 듯.